bears/general/InvalidLinkBear.py

```diff
@@ -44,8 +57,7 @@ def run(self, filename, file, timeout: int=DEFAULT_TIMEOUT):
 
         :param timeout: Request timeout period.
         '''
-        for line_number, link, code in InvalidLinkBear.find_links_in_file(
-                file, timeout):
+        for line_number, link, code in self.find_links_in_file(file, timeout):
```

Comment:
actually I think you'll rather want to do the `self.err` there and not in the get_status_code_checked function